src/rime/setup.cc

```diff
@@ -89,7 +89,11 @@ RIME_API void SetupLogging(const char* app_name,
   // Do not allow other users to read/write log files created by current
   // process.
   FLAGS_logfile_mode = 0600;
-  google::InitGoogleLogging(app_name);
+  if (google::IsGoogleLoggingInitialized()) {
```

Comment:
but `CHECK()` would cause the entire application to crash when the condition not met

https://github.com/google/glog#check-macros

Reply:
> but `CHECK()` would cause the entire application to crash when the condition not met
> 
> https://github.com/google/glog#check-macros

You are right. Checking it manually is better.